sqlglot/transforms.py

```diff
@@ -377,6 +377,16 @@ def eliminate_semi_and_anti_joins(expression: exp.Expression) -> exp.Expression:
     return expression
 
 
+def todays_to_datediff(expression: exp.Expression) -> exp.Expression:
+    if isinstance(expression, exp.ToDays):
+        return exp.DateDiff(
+            this=expression.this,
+            expression=exp.cast(exp.Literal.string("0000-01-01"), to=exp.DataType.Type.DATE),
```

Comment:
this doesn't seem right, should it be epoch?

Reply:
Nope. lol

Reply:
> Given a date date, returns a day number (the number of days since year 0). Returns NULL if date is NULL.
> 
> mysql> SELECT TO_DAYS(950501);
>        -> 728779
> mysql> SELECT TO_DAYS('2007-10-07');
>         -> 733321

Reply:
if this is mysql specific, you could just do this the transform at parse time, so to_days turns into a date diff, and then you won't have to transpile it in every other dialect